sklearn/utils/tests/test_response.py

```diff
@@ -124,24 +140,28 @@ def test_get_response_values_binary_classifier_predict_proba():
     response_method = "predict_proba"
 
     # default `pos_label`
-    y_pred, pos_label = _get_response_values(
+    results = _get_response_values(
         classifier,
         X,
         response_method=response_method,
         pos_label=None,
+        return_response_method_used=return_response_method_used,
     )
-    assert_allclose(y_pred, classifier.predict_proba(X)[:, 1])
-    assert pos_label == 1
+    assert_allclose(results[0], classifier.predict_proba(X)[:, 1])
+    assert results[1] == 1
+    if return_response_method_used:
+        assert results[2] == "predict_proba"
 
     # when forcing `pos_label=classifier.classes_[0]`
-    y_pred, pos_label = _get_response_values(
+    results = _get_response_values(
```

Comment:
I think the following should work and should be more explicit if we do not intend to inspect the potential third returned value.

```suggestion
    y_pred, pos_label, *_ = _get_response_values(
```